MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -566,13 +566,23 @@ - (void)pushEvent:(MMEEvent *)event {
 - (void)locationManager:(MMELocationManager *)locationManager didUpdateLocations:(NSArray *)locations {
     MMELOG(MMELogInfo, MMEDebugEventTypeLocationManager, ([NSString stringWithFormat:@"Location manager sent %ld locations, instance: %@", (long)locations.count, self.uniqueIdentifer.rollingInstanceIdentifer ?: @"nil"]));
     
-    for (CLLocation *location in locations) {        
-        MMEMapboxEventAttributes *eventAttributes = @{
+    for (CLLocation *location in locations) {
+        MMEMutableMapboxEventAttributes *eventAttributes = [[MMEMutableMapboxEventAttributes alloc] init];
+        
+        if ([location floor]) {
```

Comment:
Nit, but I'd add this if check after the rest of the primary attributes are added